serverless/src/getSavedAccessConfigs/handler.js

```diff
@@ -48,6 +48,7 @@ const getSavedAccessConfigs = async (event, context) => {
         collection_id: collectionId,
         access_method: accessMethod
       } = config
+      console.log(`Found savedAccessConfigs for ${accessMethod.type} collectionIds: ${collectionIds}`)
```

Comment:
Add a test for this to make sure we don't remove or change it by accident. use `spyOn(console, 'log')` and do `toHaveBeenCalledTimes()` and `toHaveBeenCalledWith()`